### ISSUE TITLE
Feature request: can ViewInspector inspect the native UIViewRepresentable to get UIView ?

### ISSUE THREAD

**Issue by dreampiggy** [nalexn/ViewInspector#7]
This is supported from another SwiftUI testing framework:

https://github.com/siteline/SwiftUI-Introspect

It use the UIKit/AppKit view hierarchy to search the view end with `ViewHost`.

Maybe we can have such a API?

```swift
func uiView<T>(_ type: T.Type) throws -> T.UIViewType
    where T: Inspectable & UIViewRepresentable

func nsView<T>(_ type: T.Type) throws -> T.NSViewType
    where T: Inspectable & NSViewRepresentable
```

**Comment by nalexn** [comment]
This is now available in the `v0.3.5`. I've added support for (UI/NS)View and (UI/NS)ViewController (`UIViewRepresentable` and `UIViewControllerRepresentable`):

```swift
view.actualView().uiView()
view.actualView().viewController()
```

**Comment by dreampiggy** [comment]
I've test the latest version and `uiView()` / `nsView()` works as expected !
Thanks for all of your effort for this feature. Now I've remove that SwiftUI-Introspect and use this one instead.

https://github.com/SDWebImage/SDWebImageSwiftUI/commit/f193c612b104085c482267252a8fc5d25d5d5e1d
